### PR TITLE
Ortho Camera Snap

### PR DESCRIPTION
**Pull request by wafflekat**
Added snap methods to ortho cam
Adhering to the keybinding stuff, though since these are specific to ortho cams I thought to indicate that but that makes the text big and ugly in the options.
Personally one of the most important features in a haven client. I keep clicking to snap and forget it's not implemented.. So here it is 😄

### CODE REVIEW

File: src/haven/MapView.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ public void rotate(Coord r) {
 	    tangl = tangl % ((float)Math.PI * 2.0f);
 	    wheel(Coord.z, 5 * r.y);
 	}
-    
+
```
Comment (wafflekat) [inline src/haven/MapView.java:164]: No idea where these are coming from. Intellij says everything's the same

**Comment by EnderWiggin** [comment]
I would prefer if this worked for all cameras.

**Comment by tomventura** [comment]
> I would prefer if this worked for all cameras.

I'll give it a shot

**Comment by wafflekat** [comment]
@EnderWiggin Added to all cameras. Note that on isometric ortho it'll snap back to isometric whenever camera is zoomed / moved. Could add a check for that but personally I'd have it on free ortho, not iso ortho. Anyway, can finally have this again.

**Pull request merged**